hw/formal/tools/vcformal/fpv_vcf.tcl

```diff
@@ -0,0 +1,163 @@
+# Copyright lowRISC contributors.
```

Comment:
I think renaming this to `fpv.tcl` should enable you move things to `common_fpv_cfg.hjson`.